exaudfclient/docs/script_options_requirments.md

```diff
@@ -0,0 +1,372 @@
+
+The parser must correctly identify and handle Script Options with the syntax `%<optionKey><white spaces><optionValue>;`.
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+### White Spaces
+`req~white-spaces~1`
+
+The parser must treat the following list of white spaces as token separator:
+|======================================================= 
+| Name         | C syntax      | ASCII Dec | ASCII Hex | 
+| tabulator    | '\t'          | 9         | 0x09      |
+| vertical tab | '\v'          | 11         | 0x0b     |
+| form feed    | '\f'          | 12         | 0x0c     |
+| space        | ' '           | 30        | 0x20      |     
+|======================================================= 
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+
+### Leading White Spaces Options Parsing
+`req~leading-white-spaces-script-options-parsing~1`
+
+The parser must accept Script Options for lines starting with any number of white space characters before the Script Options.
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+Depends:
+ - `req~white-spaces~1`
+
+### Ignore anything which is not a Script Option
+`req~ignore-none-script-options~1`
+
+If there is any character in front of a Script Option which is not a white space, the parser must ignore the option(s). 
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+Depends:
+ - `req~white-spaces~1`
+
+### Multiple Line Script Options Parsing
+`req~multiple-lines-script-options-parsing~1`
+
+The parser must recognize Script Options at any line in the given script code. This is especially important because the `%import` option (see requirement `req~java-import-option-replace-referenced-scripts~1`) might replace options with Java code in the final script code.
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+### White Spaces Options Parsing V1
+`req~white-spaces-script-options-parsing-v1~1`
+
+All white spaces between the option key and option value are to be interpreted as separator. 
+White spaces between the option value and the terminating ";" are to be removed from the option value.
+
+Needs: dsn
+
+Tags: V1
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+Depends:
+ - `req~white-spaces~1`
+
+### White Spaces Options Parsing V2
+`req~white-spaces-script-options-parsing-v2~1`
+
+All white spaces between the option key and option value are to be ignored. The following rules for escape sequences at **the start** of a script optionValue are to be applied:
+- '\ ' => space character
+- '\t' => <tab> character
+- '\f' => <form feed> character
+- '\v' => <vertical tab> character
+
+White spaces in the middle of the option value and between the option value and the terminating ";" shall be interpreted as part of the value.
+The rationale is that the new version of the parser should be as much as possible backwards compatible to V1, because it will simplify migration of existing UDF's.
```

Comment:
```suggestion

Rationale: The new version of the parser should be as much as possible backwards compatible to V1, because it will simplify migration of existing UDF's.
```